public-config.yml

```diff
@@ -10,16 +10,14 @@ binary-builder-git-uri: https://github.com/SUSE/cf-binary-builder
 cf-release-git-uri: https://github.com/cloudfoundry/cf-release
 
 buildpacks-github-org: SUSE
-buildpacks-binaries-s3-bucket: cf-opensusefs2
-buildpack-dependencies-host-domain: cf-opensusefs2.s3.amazonaws.com
 bosh-lite-domain-name: cf-app.com
 
 buildpacks-ci-pipeline-uri: http://concourse.suse.de/teams/main/pipelines/cf-buildpacks-ci
 buildpacks-ci-git-uri-public: https://github.com/SUSE/cf-buildpacks-ci
 public-buildpacks-ci-robots-git-uri-public: https://github.com/SUSE/cf-public-buildpacks-ci-robots
```

Comment:
I know this isn't part of this PR, but the `public-` prefix/suffix is sticking out like a sore thumb to me here.

Is there any reason not to `s/public-(.*)-public/$1/`?

Reply:
I'd rather not change this as it would move us further away from upstream and make it harder to rebase. In this case it's not the end of the world because it's only used in one pipeline, but in general that's my reason.

Reply:
Ok, didn't realize this name was used as-is upstream. In that case I agree with not changing it gratuitously.